framework/wazuh/core/cluster/worker.py

```diff
@@ -405,7 +214,31 @@ def get_manager(self):
         AbstractClientManager
             Worker object.
         """
-        return self.manager
+        return self.server
+
+    def setup_sync_integrity(self, sync_type: bytes, data: bytes = None) -> Tuple[bytes, bytes]:
```

Comment:
I think this name is confusing (so far Integrity is file sync, and in this function, files are not being synced but wazuh-db/strings).